test/save-in-progress/01-sip.e2e.spec.js

```diff
@@ -1,11 +1,12 @@
+const moment = require('moment');
```

Comment:
What's the difference between this test file and `01-sip-auto`?

Reply:
This is for save and finish later, the other is for auto saving. I'll change the file names to make that clearer.